resources/keycloak/create_local_cert.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+########## ONLY USE THIS SCRIPT FOR DEVELOPMENT PURPOSES ##########
+########## FOR PRODUCTION, PLEASE USE A VALID SSL CERTIFICATE ##########
+
+mkdir -p ./ssl_cert
+
+# generate self-signed ssl key and cert
+openssl req -newkey rsa:2048 -nodes -keyout ./ssl_cert/key.pem -x509 -days 3650 -out ./ssl_cert/cert.pem 
```

Comment:
Minor issue, we might want to check if openssl is installed or call it out as a dependency in the README

Reply:
Just added this to the README